webdriver/search_test.go

```diff
@@ -109,7 +109,7 @@ func getSearchElement(wd selenium.WebDriver) (selenium.WebElement, error) {
 
 func getPathPartElements(wd selenium.WebDriver, element string) ([]selenium.WebElement, error) {
 	e, err := wd.FindElement(selenium.ByTagName, "wpt-app")
-	if err != nil {
+	if e == nil || err != nil {
```

Comment:
Is this meant to happen? All the uses of `FindElement` I can see in the upstream project seem to assume that if `err == nil` then the element is non-nil (e.g. https://github.com/tebeka/selenium/blob/a49cf4b98a36c2b21b1ccb012852bd142d5fc04a/example_test.go#L80, https://github.com/tebeka/selenium/blob/e24f7b7726ca0d40f51b3ba4c5d5c4ddcb4bae80/remote.go#L749).

Reply:
Good point. "No such element" is an error: https://github.com/tebeka/selenium/blob/e24f7b7726ca0d40f51b3ba4c5d5c4ddcb4bae80/remote.go#L28

Let me take another look.

Reply:
Thanks a lot for spotting this! I indeed didn't read the stacktrace closely enough. The bug we are seeing is that `e.shadowRoot` is null, instead of `e` being null (it wouldn't).